changelog.md

```diff
@@ -10,6 +10,12 @@ class: changelog
 
 {% capture changelog %}
 
+## Version 5.1.5 December 19, 2019
+* Fixed the UI not working with hybrid devices that accept both touch and mouse events.
+* Fixed bug with pasting image URLs when paste as text is enabled.
```

Comment:
s/bug/a bug/ and s/paste as text/`paste as text`/

Reply:
Fixed